### DISCUSSION TITLE
Autocomplete input/select

### DISCUSSION THREAD

**Discussion by phoskee** [zauberzeug/nicegui#529] [Ideas / Feature Requests]
Based on my very basic knowledge (maybe I'm wrong) I noticed that there is no equivalent of [AutoCompleteCombobox](https://ttkwidgets.readthedocs.io/en/latest/examples/autocomplete/AutocompleteCombobox.html).
If i have a long list of items (especially on ios) it is difficult to search and select item. Much more convenient to start typing a few letters and hit enter, to do some stuff.
I really appreciate the work you are doing!

**Comment by falkoschindler** [comment]
Yes, that's definitely an important feature!
We're already working on it: https://github.com/zauberzeug/nicegui/issues/272

**Reply by phoskee** [reply]
my bad, thank a lot!
